.gitignore

```diff
@@ -1,4 +1,29 @@
-.idea
+# IDEA/Android Studio project
```

Comment:
Perhaps leave a guide or a link on top of this file for future maintainers to know what's the guiding principle between this file, various .gitignore files in our repo/examples etc and the .gitignore files from our creation templates.